slidez-fe-core/src/services/auth/auth-service.ts

```diff
@@ -95,7 +95,7 @@ export const isLoggedIn = () => {
 export const performLogout = () => {
     window.localStorage.removeItem(JWT)
     window.localStorage.removeItem(refreshJWT)
-    window.location.assign('/#/login')
+    window.location.assign(AppRoute.LOGIN)
```

Comment:
It seems that the problem was that we don't have '#' in AppRoute.LOGIN, so, please stay it as it is for now

Reply:
Put it back